rfcdiff.js

```diff
@@ -27,7 +27,7 @@ function change_chunk(offset) {
     if (old_chunk) {
         old_chunk.style.outline = "";
     }
-    old_chunk = new_chunk.parentNode.parentNode;
+    old_chunk = new_chunk;
```

Comment:
I hadn't noticed that there was an ID. This is fine but you could just do
"getElementById()" or at least not do $= since that's a suffix match. This is fine, though, if you like it best